### ISSUE TITLE
Accessing an "out" VEC3F by index causes a (harmless) assert

### ISSUE THREAD

**Issue by AchimStremplat** [GENIVI/ramses-logic#32]
The script
```

function interface()
    OUT.out_v = VEC3F
end

function run()
	OUT.out_v[1] = 1.0
end
```
causes in the debug build an assert to fire - if the assert is ignored, a useful error message is emitted.

Is the assert intended? I would not expect to be able to generate asserts even in debug builds by providing invalid input data?

**Comment by violinyanev** [comment]
Looks like a bug. Looking into it

**Comment by violinyanev** [comment]
Hi @AchimStremplat ! After looking into it - the code doesn't trigger an assert, it generates an error as intended. The only problem is that the error message is not very descriptive - it says "implementation error", but it should say instead "Can't assign VEC3F properties individually, must assign the entire VEC3F instead". Is this what you observed? Can you point me to the assert that you encountered?

**Comment by AchimStremplat** [comment]
@violinyanev This is what I am seeing: 
![image](https://user-images.githubusercontent.com/63643551/132691206-9ef9bd6e-d985-44c6-9735-6367927f147e.png)

**Comment by violinyanev** [comment]
Fixed in version 0.9.0!
